### PR TITLE
Misc review fixes

### PR DESCRIPTION
**Pull request by DelazJ**
contains some review feedback I couldn't make on time and some typo I found on the road

### CODE REVIEW

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 1]
```diff
@@ -5739,11 +5735,8 @@ Outputs
 
 Smooth
 ------
-Smooths the geometries in a line or polygon layer.
-It creates a new layer with the same features as the ones in the
-input layer, but with geometries containing a
-**higher number of vertices and corners** in the geometries smoothed
-out.
+Smooths the geometries in a line or polygon layer by
+adding more **vertices and corners** to the features geometries.
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]:
```suggestion
adding more **vertices and corners** to the feature geometries.
```

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 2]
```diff
@@ -6100,9 +6089,9 @@ Outputs
    * - **Split**
      - ``OUTPUT``
      - [vector: line]
-     - The new line features - all with line geometries that have a
+     - The new line vector layer - Features have geometries that have a
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]:
```suggestion
     - The new line vector layer - the length of the feature geometries
```

File: source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 3]
```diff
@@ -6100,9 +6089,9 @@ Outputs
...
        length that is less than or equal to the length specified in
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectorgeometry.rst]:
```suggestion
       is less than or equal to the length specified in
```

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 4]
```diff
@@ -509,7 +509,7 @@ You can set the :guilabel:`Maximum (inclusive)` and :guilabel:`Minimum
 be visible.
 It will be hidden outside this range.
 The |mapIdentification| :sup:`Set to current canvas scale` button helps
-you use the current map canvas scale as boundary of.
+you use the current map canvas scale as boundary of the range visibility.
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
you use the current map canvas scale as a boundary for the range visibility.
```

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 5]
```diff
@@ -1103,9 +1103,12 @@ Reverse Line
...
 cartographical purposes or when preparing for network analysis.
 
-Activate the reverse line tool by clicking |reverseLine|
-:sup:`Reverse line`.
-Select the line with the cursor to change the direction of the line.
+To change a line direction:
+
+#. Activate the reverse line tool by clicking |reverseLine|
+   :sup:`Reverse line`.
+#. Click the line with the cursor. The direction of the line
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]:
```suggestion
#. Click on the line. The direction of the line
```

**Comment by DelazJ** [comment]
> Thanks a lot, @DelazJ, and sorry about all the mistakes.

No need to be sorry @havatv This refactoring was a big work and it's natural to have some issues. And this is a team work and in perpetual improvement.

**Pull request merged**